### PR TITLE
Added missing copyright headers

### PR DESCRIPTION
**Pull request by rrati**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/718#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/apps/blackduck/components/pvc/v1/pvc.go [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
+under the License.
+*/
+
 package v1
 
 import (
 	"fmt"
 	"github.com/blackducksoftware/horizon/pkg/components"
 	blackduckapi "github.com/blackducksoftware/synopsys-operator/pkg/api/blackduck/v1"
+	"github.com/blackducksoftware/synopsys-operator/pkg/apps/blackduck"
```
Comment (msenmurugan) [inline pkg/apps/blackduck/components/pvc/v1/pvc.go]: why it is added?

File: pkg/apps/blackduck/components/pvc/v2/pvc.go [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+under the License.
+*/
+
 package v2
 
 import (
 	"fmt"
 	"github.com/blackducksoftware/horizon/pkg/components"
 	blackduckapi "github.com/blackducksoftware/synopsys-operator/pkg/api/blackduck/v1"
+	"github.com/blackducksoftware/synopsys-operator/pkg/apps/blackduck"
```
Comment (msenmurugan) [inline pkg/apps/blackduck/components/pvc/v2/pvc.go]: why it is added?

File: pkg/apps/blackduck/components/pvc/v1/pvc.go (hunk 1 above)
Comment (rrati) [reply pkg/apps/blackduck/components/pvc/v1/pvc.go]: I'm not sure.  I didn't add it so I suspect GoLand did.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/718#pullrequestreview-269210244" title="Approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/718#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/apps/blackduck/components/pvc/v1/pvc.go (hunk 1 above)
Comment (yashbhutwala) [reply pkg/apps/blackduck/components/pvc/v1/pvc.go]: It is needed, that's why travis on master is failing. I was gonna patch it in PR #704

Comment (rrati) [reply pkg/apps/blackduck/components/pvc/v1/pvc.go]: They are needed to compile, but I didn't add them and I'm not sure GoLand did either.  Maybe an issue with the version I branched from

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/718#issuecomment-516978539" title="LGTM">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/718#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**